src/utils.js

```diff
@@ -2,16 +2,25 @@
  * @ignore
  */
 export default class {
-    getPerformanceInfo() {
-        let timing = window.performance.timing,
-            interactive = timing.domInteractive - timing.domLoading,
-            dcl = timing.domContentLoadedEventStart - timing.domLoading,
-            complete = timing.domComplete - timing.domLoading;
-        return {
-            ptInteractive: interactive >= 0 ? interactive : undefined,
-            ptDcl: dcl >= 0 ? dcl : undefined,
-            ptComplete: complete >= 0 ? complete : undefined
-        };
+    getPerformanceInfo(targetWindow) {
+        if('navigation' in window[targetWindow].performance && 'timing' in window[targetWindow].performance){
+            let timing = window[targetWindow].performance.timing,
```

Comment:
Is there any reason to use `let` rather than `const`?

Reply:
fixed